src/main/resources/warnings.json

```diff
@@ -673,7 +673,8 @@
     "url": "https://jenkins.io/security/advisory/2017-04-10/",
     "versions": [
       {
-        "pattern": ".*"
+        "lastVersion": "1.5.2",
+        "pattern": "1[.][012345](|[.-].*)"
```

Comment:
To clarify, the regex is what matters to Jenkins -- `lastVersion` is just for documentation on the plugin site or wiki pages.